classes/ActionScheduler.php

```diff
@@ -110,6 +110,7 @@ public static function init( $plugin_file ) {
 
 		if ( defined( 'WP_CLI' ) && WP_CLI ) {
 			WP_CLI::add_command( 'action-scheduler', 'ActionScheduler_WPCLI' );
+			require_once( self::plugin_path('deprecated/ActionScheduler_WPCLI_Scheduler_command.php') );
```

Comment:
If you change the class name to `ActionScheduler_WPCLI_Scheduler_Command_Deprecated` then the autoloader will load it automatically if the class is used.

Reply:
@rrennick This change is from #267 (please see https://github.com/Prospress/action-scheduler/pull/267#issuecomment-477174034).

Reply:
The code hasn't been released yet so items that were missed in the first review can be fixed without negative impact.

Reply:
@rrenick The code hasn't even been merged yet, which is why it appears in this PR, even though it's not related to this PR.

Additionally, changing the class name will break backwards compatibility (for direct references from other sources).

Reply:
Either the autoloader could be updated or a stub could be added that mitigated the issue. Currently, the only `require`s for classes is in the main plugin file. It would be good to keep it that way.

Reply:
Moving convo to https://github.com/Prospress/action-scheduler/pull/267#issuecomment-479562559.